packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -659,6 +659,10 @@ export default defineComponent({
     justify-content: center;
     align-content: center;
   }
+
```

Comment:
Remove this from select and move it to VaInput instead.
![image](https://user-images.githubusercontent.com/23530004/143912282-c2887570-ff79-4eb1-b49b-9b473104864e.png)


```suggestion
```

Reply:
add to va-input:
![code](https://user-images.githubusercontent.com/55198465/144051050-c4e52bdc-d6cf-446d-9dcd-aa30f3c2488a.png)

remove div-wrapper from va-select:
![code](https://user-images.githubusercontent.com/55198465/144050904-72122066-cd64-4ce5-a889-6ada1bf6b3a5.png)